posydon/popsyn/synthetic_population.py

```diff
@@ -725,10 +834,11 @@ def select(self, where=None, start=None, stop=None, columns=None):
         # Select specific columns
         >>> df = Oneline.select(columns=['S1_mass_i', 'S1_mass_f'])
         """
-        with pd.HDFStore(self.filename, mode="r") as store:
-            return store.select(
-                "oneline", where=where, start=start, stop=stop, columns=columns
-            )
+        return super().select(key='oneline',
```

Comment:
I was trying to tests the .select functionality and couldn't get `pop.oneline.select(where="S2_mass_i > 30")` to work. This is an example given on line 829. Am I missing something? 

The error message said it "contains an invalid variable reference ...The currently defined references are: index,columns,state_i,state_f,event_i,event_f,step_names_i,step_names_f,S1_state_i,S1_state_f,S2_state_i,S2_state_f,S1_SN_type,S2_SN_type,interp_class_HMS_HMS,interp_class_CO_HeMS,interp_class_CO_HMS_RLO,interp_class_CO_HeMS_RLO,mt_history_HMS_HMS,mt_history_CO_HeMS,mt_history_CO_HMS_RLO,mt_history_CO_HeMS_RLO
"

We should add it to the defined references or remove that example from the documentation.

Reply:
That came from the switch from using `data_columns=True` to  `data_columns=False`. It removes the option to use `where=" "` non-string variables.
This was the choice we made when we wanted to have faster file merging after a population run. 

However, the info in the documentation should indeed be removed. I have removed the example from the docstring and added a statement in the docstring on what you can search. It should not be in the tutorial anymore. There is should explicitly state that it can only select on string columns.